### ISSUE TITLE
CLI not working anymore. missing @graphql-inspector/commands

### ISSUE THREAD

**Issue by bfagundez** [kamilkisiela/graphql-inspector#1392]
The command asks for that package now:
```
$ graphql-inspector validate documents.graphql src/schema.graphql
internal/modules/cjs/loader.js:589
    throw err;
    ^

Error: Cannot find module '@graphql-inspector/commands'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:587:15)
    at Function.Module._load (internal/modules/cjs/loader.js:513:25)
    at Module.require (internal/modules/cjs/loader.js:643:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/usr/local/share/.config/yarn/global/node_modules/@graphql-inspector/cli/index.js:7:18)
    at Module._compile (internal/modules/cjs/loader.js:707:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:718:10)
    at Module.load (internal/modules/cjs/loader.js:605:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:544:12)
    at Function.Module._load (internal/modules/cjs/loader.js:536:3)
```
It was working this morning.

**Comment by kamilkisiela** [comment]
Oh I broke the build... Released a new version but with a broken internal tool. Give me few minutes

**Comment by kamilkisiela** [comment]
@bfagundez try now, I released `v1.30.3`
